Makefile

```diff
@@ -46,7 +46,7 @@ build: ## Build binary
 
 .PHONY: container-image
 container-image: ## Build container image
-	docker build -t ${CONTAINER_IMAGE_REF} .
+	docker build -t ${CONTAINER_IMAGE_REF} . --platform linux/amd64
```

Comment:
I don't think we need to specify a target platform here. If you would like to do that, you will also need to update the github actions to build and release those images too.

Reply:
This was just something i needed to do to build it locally as i am using a mac

Reply:
You can use env to set the default platform via `export DOCKER_DEFAULT_PLATFORM=linux/amd64`

Reply:
You can remove this from the makefile